src/main/java/uk/gov/justice/hmpps/prison/api/resource/AdjudicationsResource.java

```diff
@@ -117,4 +121,41 @@ public AdjudicationDetail getAdjudication(
     public List<AdjudicationDetail> getAdjudications(@Parameter(description = "The adjudication numbers", required = true, example = "[1,2,3]") @RequestBody final List<Long> adjudicationNumbers) {
         return adjudicationsService.getAdjudications(adjudicationNumbers);
     }
+
+    @ApiResponses({
+        @ApiResponse(responseCode = "201", description = "Created"),
+        @ApiResponse(responseCode = "403", description = "The client is not authorised for this operation"),
+        @ApiResponse(responseCode = "400", description = "Invalid request - ie missing hearing location or date", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
+        @ApiResponse(responseCode = "404", description = "No match was found for the adjudication number", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))})
+    })
+    @Operation(summary = "Creates an OIC hearing", description = "Requires MAINTAIN_ADJUDICATIONS access and write scope")
+    @PostMapping("/adjudication/{adjudicationNumber}/hearing")
+    @ProxyUser
+    @PreAuthorize("hasRole('MAINTAIN_ADJUDICATIONS') and hasAuthority('SCOPE_write')")
+    public ResponseEntity<OicHearingResponse> createOicHearing(
+        @Valid @RequestBody @Parameter(description = "OIC hearing to save", required = true) final OicHearingRequest oicHearingRequest,
+        @PathVariable("adjudicationNumber") final Long adjudicationNumber
+        ) {
+        var oicHearing = adjudicationsService.createOicHearing(adjudicationNumber, oicHearingRequest);
+        return ResponseEntity
```

Comment:
very minor style observation: but you good as used the response annotation to specify the 201 and just returned a plain java object rather than a ResponseEntity